tensorboard/plugins/profile/tf_op_profile/tf-op-profile.html

```diff
@@ -34,41 +34,26 @@
 tf-op-details {
   position: fixed;
   /* don't set top, so it ends up next to tf-op-table */
+  padding-top: 6.5em;
   left: 16px;
   width: 330px;
 }
 :host {
   display: block;
   margin-right: 1.5em;
+}
+.description {
+  margin-bottom: 2em;
 }
     </style>
     <div class="tf-op-profile">
-      <h4>Overall TPU FLOPS utilization is
+      <h3>Overall TPU FLOPS utilization is
         <span style$="color:[[_textColor(_root)]]">[[_utilizationPercent(_root)]]</span>
-      </h4>
-      <p>Modifying your model's architecture, data dimensions, and improving the
-      efficiency of CPU operations may help reach the TPU's FLOPS potential.</p>
-      <h4>The most time was spent in a <b>[[_costlyOp.xla.category]] operation</b>
-        ([[_percent(_costlyOp.metrics.time)]])</h4>
-      <p hidden$="[[_hasFlops(_costlyOp)]]">
-        [[_costlyOp.name]] is
-        <span style$="color:[[_textColor(_costlyOp)]];">overhead</span>,
-        and a good target for optimization.
-      </p>
-      <!-- extra div to avoid HTML parsing bugs in polymer/vulcanize -->
-      <div hidden$="[[!_hasFlops(_costlyOp)]]">
-      <p>
-      While active, <code>[[_costlyOp.name]]</code> uses
-        <span style="color:[[_textColor(_costlyOp)]]">[[_utilizationPercent(_costlyOp)]]</span>
-        of the computational potential of the chip.
-        <span hidden$="[[_goodFlops(_costlyOp)]]">
-          This is a good target for optimization.
-        </span>
-        <span hidden$="[[!_goodFlops(_costlyOp)]]">
-          This is pretty good - other operations may be better targets for optimization.
-        </span>
-      </p>
-      </div>
+      </h3>
+      <div class="description">
+      <p>Modifying your model's architecture, data dimensions, and improving
+      the efficiency of CPU operations may help reach the TPU's FLOPS potential.
+      </p></div>
```

Comment:
nit: </div> goes on next line.

Reply:
Done. Set the description width to 600px.